Fix `TestTransform.testRepr`

## 🐛 Bug

<!-- A clear and concise description of what the bug is. -->

## To Reproduce

Steps to reproduce the behavior:

1. `inv test`
1.
1.

<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

## Environment

 - OS (e.g., Linux):
 - Python version:
 - Any other relevant information:

## Additional context

```
_______________________________________________ TestTransform.testRepr ________________________________________________

self = <test.unit.flaris.test_transform.TestTransform object at 0x000002ABD39192C8>

    def testRepr(self):
        transform = Transform(position=Vector(0, 0, 0),
                              rotation=Vector(0, 0, 0),
                              scale=Vector(1, 1, 1))

        actual = repr(transform)
        expected = ("Transform(position=Vector(0, 0, 0), "
                    "rotation=Vector(0, 0, 0), scale=Vector(1, 1, 1))")

>       assert actual == expected
E       AssertionError: assert 'Transform(sc...tor(1, 1, 1))' == 'Transform(pos...tor(1, 1, 1))'
E         - Transform(scale=Vector(1, 1, 1))
E         + Transform(position=Vector(0, 0, 0), rotation=Vector(0, 0, 0), scale=Vector(1, 1, 1))

test\unit\flaris\test_transform.py:122: AssertionError
```

--- a/flaris/transform.py
+++ b/flaris/transform.py
@@ -208,10 +208,12 @@ class Transform(Component):  # pylint: disable=unused-argument
             string += f"position={self.position}, "
         if self.rotation != Vector(0, 0, 0):
             string += f"rotation={self.rotation}, "
-        if self.scale != Vector(0, 0, 0):
+        if self.scale != Vector(1, 1, 1):
             string += f"scale={self.scale}, "
 
         # Remove trailing ', '
-        string = string[:-len(", ")]
+        if ", " in string:
+            string = string[:-len(", ")]
+
         string += ")"
         return string

--- a/test/unit/flaris/test_transform.py
+++ b/test/unit/flaris/test_transform.py
@@ -117,8 +117,7 @@ class TestTransform:
         (Transform(position=Vector(1, 0, 0),
                    rotation=Vector(30, 45, 60),
                    scale=Vector(2, 2, 2)),
-         "Transform(position=Vector(1, 0, 0), rotation=Vector(30, 45, 60), scale=Vector(2, 2, 2))"
-        )
+         "Transform(position=Vector(1, 0, 0), rotation=Vector(30, 45, 60), scale=Vector(2, 2, 2))")  # noqa: E501
     ])
     def testRepr(self, transform, expected_string):
         assert repr(transform) == expected_string

--- a/test/unit/flaris/test_transform.py
+++ b/test/unit/flaris/test_transform.py
@@ -110,6 +110,7 @@ class TestTransform:
         transform.rotate(angles)
         assert transform.rotation == expected_rotation
 
+    # pylint: disable=line-too-long
     @pytest.mark.parametrize("transform, expected_string", [
         (Transform(), "Transform()"),
         (Transform(position=Vector(1, 0, 0)),

--- a/test/unit/flaris/test_transform.py
+++ b/test/unit/flaris/test_transform.py
@@ -110,13 +110,15 @@ class TestTransform:
         transform.rotate(angles)
         assert transform.rotation == expected_rotation
 
-    def testRepr(self):
-        transform = Transform(position=Vector(0, 0, 0),
-                              rotation=Vector(0, 0, 0),
-                              scale=Vector(1, 1, 1))
-
-        actual = repr(transform)
-        expected = ("Transform(position=Vector(0, 0, 0), "
-                    "rotation=Vector(0, 0, 0), scale=Vector(1, 1, 1))")
-
-        assert actual == expected
+    @pytest.mark.parametrize("transform, expected_string", [
+        (Transform(), "Transform()"),
+        (Transform(position=Vector(1, 0, 0)),
+         "Transform(position=Vector(1, 0, 0))"),
+        (Transform(position=Vector(1, 0, 0),
+                   rotation=Vector(30, 45, 60),
+                   scale=Vector(2, 2, 2)),
+         "Transform(position=Vector(1, 0, 0), rotation=Vector(30, 45, 60), scale=Vector(2, 2, 2))"
+        )
+    ])
+    def testRepr(self, transform, expected_string):
+        assert repr(transform) == expected_string

